Gruntfile.js

```diff
@@ -34,6 +66,62 @@ module.exports = function (grunt) {
             dist: {
                 src: ['components/fc.js', 'components/utilities/*.js', '<%= meta.componentsJsFiles %>'],
                 dest: 'dist/<%= pkg.name %>.js'
+            },
+            visualTests: {
+                options: {
+                    sourceMap: true
+                },
+                src: 'visual-tests/src/site/assets/js/**/*.js',
+                dest: 'visual-tests/dist/assets/index.js',
+            }
+        },
+
+        copy: {
```

Comment:
Ok sounds like it'll be snappy then, we can always revisit if we think it's getting slow.